.circleci/config.yml

```diff
@@ -74,7 +108,9 @@ workflows:
   version: 2
   build_and_test:
     jobs:
+      - build_linux_c_compilergccpython3.5
```

Comment:
These won't work, as we have not built any of the deps with the modern compilers for python 3.5